hr_timesheet_sheet/models/account_analytic_line.py

```diff
@@ -12,6 +12,12 @@ class AccountAnalyticLine(models.Model):
         comodel_name='hr_timesheet.sheet',
         string='Sheet',
     )
+    negative_unit_amount = fields.Boolean(
```

Comment:
Still I don't understand why you need an option to prevent negative amounts. This option could be in a separate dedicated module.
Anyway the standard Odoo allows negative amounts so this module should do the same by default.

Reply:
I'd support @astirpe in this position, it should be a separate module. Probably as well as limitation of changing employee company

Reply:
The problem is the methods that use this. They are difficult to write in a way to do easy supers...

Reply:
I've looked through the code - I believe a refactor would help that, code pieces that use `negative_unit_amount` are a bit similar, and maybe can be extracted as `_compute_unit_amount` or something

Reply:
ok, done now in a way that can be easy superseded. And also, as @astirpe said, now it's enabled the negative amounts by default.